src/lib/consts.ts

```diff
@@ -5,3 +5,19 @@ export const DEFAULT_SHIBUYA_NETWORK_URL = "wss://shibuya.public.blastapi.io";
 
 export const ARTIFACTS_PATH = "artifacts";
 export const TYPED_CONTRACTS_PATH = "typedContracts";
+
+export const DEFAULT_NODE_INFO = {
```

Comment:
To make it more easy to maintain use the swankyNodeVersions map:

`export const DEFAULT_NODE_INFO = swankyNodeVersions.get("1.6.0");`